{
  "FaultLabel": "Resolved power cut",
  "Type": "ResolvedFault",
  "Icon": "icon-success",
  "ShowViewOnMapCTA": false,
  "UnderReview": false,
  "AdditionalFaultInfo": "Good news! The electricity in your area has been restored. The power cut was caused by an unexpected incident on our underground cable that provides your supply.",
  "CTAs": [],
  "faultNumber": "10629090",
  "multipleFaultNumbers": null,
  "date": "2020-12-06T19:20:39",
  "region": "Ashton-Under-Lyne",
  "faultType": "Resolved Fault",
  "faultStatus": "Resolved",
  "previousFaultStatus": "Resolved",
  "consumersOff": 0,
  "estimatedTimeOfRestoration": null,
  "estimatedTimeOfRestorationMajority": "2020-12-06T19:20:39",
  "information": "",
  "actualTimeOfRestoration": "2020-12-06T19:20:39",
  "outageCentrePoint": {
    "lat": 53.521158,
    "lng": -2.04685
  },
  "outageLocations": null,
  "AffectedPostcodes": " OL5 0NY, OL5 0PF, OL5 0PG, OL5 0PL, OL5 0PP, OL5 0PQ, OL5 0PR, OL5 0QH, OL5 0QQ, OL5 0QX",
  "EtrMUnderReview": true,
  "WebTMSFaultType": "U/GM"
}